### ISSUE TITLE
"Bypass" method doesn't work if an user has a space in its name

### ISSUE THREAD

**Issue by Tortillas-IT** [EbolaMan-YT/UAC-Bypass#4]
title

**Comment by ghost** [comment]
womp womp

**Comment by l-urk** [comment]
yeah I actually solved this problem if you people want the code for that

**Comment by Tortillas-IT** [comment]
> yeah I actually solved this problem if you people want the code for that

send the repo, or make a pull request, please

**Comment by l-urk** [comment]
done. please review change.

**Comment by l-urk** [comment]
The owner isn't adding the change so I forked the repo and updated the file here.
https://github.com/l-urk/UAC-Bypass/
This will work if the user has a space in the name.
